src/scout_apm/core/context.py

```diff
@@ -1,14 +1,17 @@
 # coding=utf-8
 from __future__ import absolute_import, division, print_function, unicode_literals
 
-import threading
-
 from scout_apm.core.tracked_request import TrackedRequest
 
+try:
+    from asgiref.local import Local
+except ImportError:
+    from threading import local as Local
```

Comment:
So if they are not running Python 3.5+ with asgiref, we fall back to using Treading.local, which will not support async tracking, correct?

I wonder if we should log a line that syas something to that effect: `Async tracking unavailable (Requires Python 3.5+)` or similar

Reply:
It's only in Python 3.5 that the async functionality was added to Python so that wouldn't really add any information :)

Reply:
Ah, I had thought Python had async for longer. Nevermind, then!

Reply:
There's been gevent and other styles of async for a while - in general it's a confusing landscape. Still need to get around to official gevent support in #28.